lib/util/commandSplit.js

```diff
@@ -0,0 +1,37 @@
+const conf = require(__dirname + '/conf');
+
+function commandSplit(com) {
+  let command, wikiaDomain, wikiaSearch;
```

Comment:
I meant to make it more command splitting, so I can either make it more wikia specific or remove the wikia parts. @vorpalhex thoughts?